tests/api_tests/base/test_serializers.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from nose.tools import *  # flake8: noqa
+
+from tests.base import ApiTestCase
+from tests import factories
+
+from api.base.settings.defaults import API_BASE
+from api.nodes.serializers import NodeSerializer
+
+class TestApiBaseSerializers(ApiTestCase):
+
+    def setUp(self):
+        super(TestApiBaseSerializers, self).setUp()
+
+        self.node = factories.ProjectFactory(is_public=True)
+
+        for i in range(5):
+            factories.ProjectFactory(is_public=True, parent=self.node)
+
+        self.url = '/{}nodes/{}/'.format(API_BASE, self.node._id)
+
+    def test_counts_not_included_in_link_fields_by_default(self):
+
+        res = self.app.get(self.url)
+        relationships = res.json['data']['relationships']
+        for relation in relationships.values():
+            link = relation['links'].values()[0]
+            assert_not_in('count', link.get('meta', {}))
```

Comment:
More specifically, I guess, is there a currently supported case where the meta field should not exist in our relationship links?